mcs/class/Makefile

```diff
@@ -105,6 +105,8 @@ monodroid_dirs := \
 	$(mobile_common_dirs)	\
 	Mono.CompilerServices.SymbolWriter	\
 	Mono.Btls.Interface	\
+	System.Runtime.Serialization.Formatters.Soap	\
+	System.Runtime.Remoting	\
```

Comment:
I'd prefer not to add it, if nobody complained in 5 years or so why to do the extra work and lock ourself into supporting this forever